src/TuDa.CIMS.Shared/Dtos/UpdateSubstanceDto.cs

```diff
@@ -10,7 +11,7 @@ namespace TuDa.CIMS.Shared.Dtos;
 public abstract record UpdateSubstanceDto : UpdateAssetItemDto
 {
     /// <summary>
-    /// An unique identifier for the chemical.
```

Comment:
I think "an unique" was correct

Reply:
I thought so too, but the IDE said it's correct like that. The web says it too